Testscripts/Windows/SETUP-RemoveIsoFromDvd.ps1

```diff
@@ -57,10 +56,35 @@ function Main {
     }
 
     if (-not $?) {
-        "Error: Unable to remove the .iso from the DVD!"
+        Write-LogErr "Unable to remove the .iso from the DVD!"
         return $False
     }
 
+    # Get Hyper-V VHD path
+    $obj = Get-WmiObject -ComputerName $hvServer -Namespace "root\virtualization\v2" -Class "MsVM_VirtualSystemManagementServiceSettingData"
+    $defaultVhdPath = $obj.DefaultVirtualHardDiskPath
+    if (-not $defaultVhdPath) {
+        Write-LogErr "Unable to determine VhdDefaultPath on Hyper-V server ${hvServer}"
+        return $False
+    }
+    if (-not $defaultVhdPath.EndsWith("\")) {
+        $defaultVhdPath += "\"
+    }
+
+    # Remove the .iso file
+    $isoPath_tmp = $defaultVhdPath + "${vmName}_CDtest.iso"
+    $isoPath = $isoPath_tmp.Replace(':','$')
+    $isoPath = "\\" + $HvServer + "\" + $isoPath
+
+    if (Test-Path "${isoPath}") {
+        try {
+            Remove-Item "${isoPath}"
+        }
+        catch {
```

Comment:
} catch {

Reply:
@lixuemin2016 Please fix the last comment, then I can merge.

Reply:
Updated, thank you so much for your review and comment.

Reply:
@lixuemin2016 Thanks for the fix.